src/Exthost/LanguageInfo.re

```diff
@@ -159,15 +172,19 @@ let getTreesitterPathFromScope = (li: t, scope: string) => {
   li.scopeToTreesitterPath |> StringMap.find_opt(scope) |> Option.join;
 };
 
-let _getLanguageTuples = (lang: Contributions.Language.t) => {
+let getLanguageTuples = (lang: Contributions.Language.t) => {
   List.map(extension => (extension, lang.id), lang.extensions);
 };
 
-let _getGrammars = (extensions: list(Scanner.ScanResult.t)) => {
```

Comment:
(Dropped the `_` from the other functions since this file has an interface file so)